docs/sdk/py/client/index.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: Client Module
+description: API reference for the Client Module Module of the ReductStore Client SDK for Python.
+---
+<head>
+    <link rel="canonical" href="https://www.reduct.store/docs/sdk/py/client" />
+</head>
+
+# Client Module
+
+<a id="reduct.client"></a>
+
+# reduct.client
```

Comment:
because of "# reduct.client" you get two h1 tags per page. Let's keep "# Client Module" and change to something like "Client module for ReductStore HTTP API (**reduct.client**)"